grammar/FuncTestCaseParser.g4

```diff
@@ -201,10 +207,27 @@ listArg
     : literalList DoubleColon listType
     ;
 
+lambdaArg
+    : literalLambda DoubleColon funcType
+    ;
+
 literalList
     : OBracket (literal (Comma literal)*)? CBracket
     ;
 
+literalLambda
+    : OParen lambdaParameters Arrow lambdaBody CParen
+    ;
+
+lambdaParameters
+    : Identifier                                    # singleParam
+    | OParen Identifier (Comma Identifier)+ CParen  # tupleParams
+    ;
+
+lambdaBody
+    : identifier OParen arguments CParen
+    ; // For now, we only allow lambda bodies of the form <func_name>(<arg1>,<arg2>,...)
```

Comment:
```suggestion
    ; // For now, we only allow lambda bodies of the form func_name(arg1,arg2,...)
```
This is probably pedantic and opinionated but I got confused thinking the angle brackets were part of the syntax (they are part of the syntax for the `funcType` after all)